src/AdoNet/OrleansSQLUtils/OrleansSQLUtils.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.OrleansSQLUtils</PackageId>
+    <Title>Microsoft Orleans SQL Utilities</Title>
+    <Description>Library of utility types for relational storage of Microsoft Orleans.</Description>
```

Comment:
The other descriptions do not have '.' here, should this one? I don't know how it has been.